gpii/node_modules/pouchManager/configs/gpii.pouchManager.config.base.json5

```diff
@@ -18,8 +18,8 @@
                     "data": [
                         "%gpii-universal/testData/dbData/clientCredentials.json",
                         "%gpii-universal/testData/dbData/gpiiAppInstallationClients.json",
-                        "%gpii-universal/build/dbData/gpiiKeys.json",
-                        "%gpii-universal/build/dbData/prefsSafes.json",
+                        "%gpii-universal/build/dbData/user/gpiiKeys.json",
```

Comment:
pouchManager is used when running GPII locally, which would be better to use the data set same as the production data set. That's why here should load snapset data from `build/dbData/snapset` instead of user data.

Reply:
oops.  switched to use "snapset".